app/src/main/java/com/arctouch/movies/base/presenter/BasePresenter.java

```diff
@@ -0,0 +1,52 @@
+package com.arctouch.movies.base.presenter;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.CallSuper;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+/**
+ * Created by marcelo.alves on 06/09/17.
+ */
+
+public abstract class BasePresenter {
+
+    protected BasePresenter() {
```

Comment:
Why did you create this protected empty constructor?

Reply:
Yes, for final version of this project it is unnecessary. This class was created on first project structure (as several other class and resources)